leakcanary-android-core/src/main/java/leakcanary/internal/DisplayLeakAdapter.kt

```diff
@@ -160,13 +160,17 @@ internal class DisplayLeakAdapter constructor(
 
     val styledClassName = styledClassSimpleName()
     var htmlString =
-      if (packageEnd != -1) "${
-        extra(
-          className.substring(
-            0, packageEnd
+      if (packageEnd != -1) {
```

Comment:
Any idea what all these code changes are about? The new code doesn't always look better.